### ISSUE TITLE
Android resources not found

### ISSUE THREAD

**Issue by danieeis** [dansiegel/Mobile.BuildTools#198]
# Description

When compile in android, occurs errors on build
```
Target _UpdateAndroidResgen:
    Target _UpdateAndroidResgenAapt2:
        Resources/drawable/splash_screen.xml(2): error APT2260: resource drawable/buildtoolslogo (aka com.jrestrepo.florida_lottery:drawable/buildtoolslogo) not found.
        Resources/mipmap-anydpi-v26/icon.xml(2): error APT2260: resource mipmap/launcher_foreground (aka com.jrestrepo.florida_lottery:mipmap/launcher_foreground) not found.
        Resources/mipmap-anydpi-v26/icon_round.xml(2): error APT2260: resource mipmap/launcher_foreground (aka com.jrestrepo.florida_lottery:mipmap/launcher_foreground) not found.
        /Library/Frameworks/Mono.framework/External/xbuild/Xamarin/Android/Xamarin.Android.Aapt2.targets(164,3): error APT2061: failed linking file resources.
    Done building target "_UpdateAndroidResgenAapt2" in project "FloridaLottery.Android.csproj" -- FAILED.
Done building target "_UpdateAndroidResgen" in project "FloridaLottery.Android.csproj" -- FAILED.
```

The android/icon.json and banner.json files is correct names and in the obj folders /Resources/ found the resources that failed linking

## Reproduction Steps

1. Configuring a new project similar the sample app of buildtools
## Environment

- OS: [OSX]
- IDE: [VSMac]
- BuildTools Version: [2.x.x]
- IDE Version: [8.8.8]
- Build Target Framework: [Xamarin.Android]

## Reproduction App

[LotterySampleApp](https://github.com/danieeis/lotteryapp)

**Comment by bijington** [comment]
@dansiegel I am not convinced my changes from #205 will have solved this one

**Comment by dansiegel** [comment]
from testing the CI build looks like you're right @bijington.... will have to figure out why the resources aren't getting included

**Comment by dansiegel** [comment]
this should be fixed by #222... and is now available in the latest preview build on NuGet
